utilities/xmlrpcpp/src/XmlRpcClient.cpp

```diff
@@ -448,7 +458,6 @@ XmlRpcClient::readHeader()
   return true;    // Continue monitoring this source
 }
 
-    
```

Comment:
Nitpick: please avoid unrelated changed to keep the diff minimal.

Reply:
Addressed with 4afa09b44e7419ee63cce7e171cdd0d411b149af